Vacation-STUDENT_ID_300506334/apps/vacations_front/src/store/services/axiosInterceptor.ts

```diff
@@ -0,0 +1,31 @@
+import axios, { AxiosError, AxiosInstance, AxiosResponse } from "axios";
+import { getTokenLS } from "../reducers/helpers/LSmanagement";
+
+
+const axiosInstance: AxiosInstance = axios.create({ baseURL: "http://localhost:3500" });
+
+axiosInstance.interceptors.request.use((request: any) => {
+  request.headers.authorization =  getTokenLS();
+  return request;
+});
+
+axiosInstance.interceptors.response.use(
```

Comment:
this function should return promise

Reply:
yes yes I missed that , the error from the response interceptor needs to
return a promise reject. I assume this gave me a lot of bugs which only now
I know what is the source for them. will fix thanks.